### PR TITLE
[chore] [exporter/clickhouse] Upgrade Logs status to Beta + README

### PR DESCRIPTION
**Pull request by SpencerTorres**
**Description:**

*NOTE: Depends on merging of #33611, #33614, and #33648*

Logs configuration is unlikely to change further with the new recommendation of [manual schema config](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/exporter/clickhouseexporter#schema-management).
The logs component of the exporter has proven stability, as seen in deployments such as [ClickHouse's Loghouse](https://clickhouse.com/blog/building-a-logging-platform-with-clickhouse-and-saving-millions-over-datadog).

With these points in mind, I suggest we upgrade the status to `beta` for logs. Traces and metrics can remain `alpha` for now.

**Documentation:**
- Updated metadata.yaml
- Updated README
- Small unrelated change: added/refactored a section that covers tool visualization

### CODE REVIEW

File: exporter/clickhouseexporter/README.md [hunk 1]
```diff
@@ -29,14 +31,16 @@ as [ClickHouse document says:](https://clickhouse.com/docs/en/introduction/perfo
...
-   Support time-series graph, table and logs.
+#### Official ClickHouse Grafana Plugin
 
-2. Analyze logs via powerful clickhouse SQL.
+The official [Grafana ClickHouse Datasource](https://grafana.com/grafana/plugins/grafana-clickhouse-datasource/) contains features that integrate directly with this exporter.
+You can view associated [Logs](https://clickhouse.com/docs/en/integrations/grafana/query-builder#logs) and [Traces](https://clickhouse.com/docs/en/integrations/grafana/query-builder#traces), as well as visualize other queries such as tables and time series graphs. Learn how to configure the OpenTelemetry integration [here](https://clickhouse.com/docs/en/integrations/grafana/config#opentelemetry).
+
+#### Grafana Altinity plugin for ClickHouse
+
+If the official plugin doesn't meet your needs, you can try the [vertamedia-clickhouse-datasource](https://grafana.com/grafana/plugins/vertamedia-clickhouse-datasource/) by Altinity, which also supports a wide range of features.
```
Comment (crobert-1) [inline exporter/clickhouseexporter/README.md:43]:
```suggestion
If the official plugin doesn't meet your needs, you can try the [Altinity plugin for ClickHouse
](https://grafana.com/grafana/plugins/vertamedia-clickhouse-datasource/), which also supports a wide range of features.
```
Nit, feel free to drop if it's not helpful.

File: exporter/clickhouseexporter/README.md [hunk 2]
```diff
@@ -29,14 +31,16 @@ as [ClickHouse document says:](https://clickhouse.com/docs/en/introduction/perfo
...
 
-1. Use [Grafana Clickhouse datasource](https://grafana.com/grafana/plugins/grafana-clickhouse-datasource/) or
-   [vertamedia-clickhouse-datasource](https://grafana.com/grafana/plugins/vertamedia-clickhouse-datasource/) to make
-   dashboard.
-   Support time-series graph, table and logs.
+#### Official ClickHouse Grafana Plugin
 
-2. Analyze logs via powerful clickhouse SQL.
+The official [Grafana ClickHouse Datasource](https://grafana.com/grafana/plugins/grafana-clickhouse-datasource/) contains features that integrate directly with this exporter.
+You can view associated [Logs](https://clickhouse.com/docs/en/integrations/grafana/query-builder#logs) and [Traces](https://clickhouse.com/docs/en/integrations/grafana/query-builder#traces), as well as visualize other queries such as tables and time series graphs. Learn how to configure the OpenTelemetry integration [here](https://clickhouse.com/docs/en/integrations/grafana/config#opentelemetry).
```
Comment (crobert-1) [inline exporter/clickhouseexporter/README.md:39]:
```suggestion
You can view associated [logs](https://clickhouse.com/docs/en/integrations/grafana/query-builder#logs) and [traces](https://clickhouse.com/docs/en/integrations/grafana/query-builder#traces), as well as visualize other queries such as tables and time series graphs. Learn how to configure the OpenTelemetry integration [here](https://clickhouse.com/docs/en/integrations/grafana/config#opentelemetry).
```
Nit

**Review by crobert-1 — COMMENTED** [review]
Usually we include a changelog entry for stability changes in components. Is there any reason we shouldn't include one to let users know that logs are now considered `beta`?

Thanks for docs updates as well, very helpful to have all of the links 👍

**Comment by SpencerTorres** [comment]
Added changelog entry + fits nits. Thanks for the review @crobert-1 !

**Review by crobert-1 — COMMENTED** [review]
Just a couple more nits.

**Comment by SpencerTorres** [comment]
No problem! Resolved the other nits. Let me know if anything else stands out 👍

**Review by crobert-1 — APPROVED** [review]

**Review by codeboten — APPROVED** [review]
Pinging other code owners to confirm: @hanjm @dmitryax @Frapschen

**Review by Frapschen — APPROVED** [review]

**Review by hanjm — APPROVED** [review]

**Review by dmitryax — APPROVED** [review]

**Comment by dmitryax** [comment]
I'd say we should remove the deprecated option before pushing it to beta: https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/33648

**Comment by SpencerTorres** [comment]
Let me know if this is good to merge now 👍

**Comment by crobert-1** [comment]
From what I can tell all of the related issues that were blocking this have been completed, so I'm adding the `ready to merge` label.

**Pull request merged**